genomic_data_service/region_indexer.py

```diff
@@ -274,9 +273,56 @@ def fetch_datasets(files, datasets):
         fetch_documents(dataset)
         datasets[dataset["accession"]] = dataset
 
+def get_encode_accessions():
+    # get file accessions for all the files using hg19 and two reference SNPs files from pickle
+    encode_accessions = list(pickle.load(open(ENCODE_ACCESSIONS_HG19_PATH, "rb")))
+    
+    # get files in experiment TF ChIP-seq using assembly GRCh38
+    endpoint = "https://www.encodeproject.org/search/?type=Experiment&control_type!=*&status=released&assay_title=TF+ChIP-seq&assembly=GRCh38&field=files.accession&field=files.preferred_default&field=files.file_format&field=files.analyses.@id&field=default_analysis&format=json&limit=all"
+    experiments = requests.get(endpoint).json()["@graph"]
+    # get files in experiment DNase-seq using assembly GRCh38
+    endpoint = "https://www.encodeproject.org/search/?type=Experiment&control_type!=*&status=released&assay_title=DNase-seq&&assembly=GRCh38&field=files.accession&field=files.preferred_default&field=files.file_format&field=files.analyses.@id&field=default_analysis&format=json&limit=all"
+    experiments = experiments + requests.get(endpoint).json()["@graph"]
+    
+    # get files in footprints
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=footprints&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = requests.get(endpoint).json()["@graph"]
+    # get files in PWMs
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=PWMs&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = annotations + requests.get(endpoint).json()["@graph"]
+    # get files in eQTLs
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=eQTLs&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = annotations + requests.get(endpoint).json()["@graph"]
```

Comment:
I think it would be better to put these into constants. Reassigning variables in general is a dangerous thing to do (here this danger just disappears by using the constants). Even better might be to make this function more focused and fetch one set at a time and store the list of accessions outside the function and update it one by one? Might be easier to catch some unexpected changes in the search results this way. In addition if we want to change something, you only need to update a constant instead of the function (possibly breaking some unrelated tests).